torch/nn/parameter.py

```diff
@@ -100,6 +100,7 @@ class UninitializedTensorMixin:
         torch.Tensor.to,
         torch.Tensor.get_device,
         torch._has_compatible_shallow_copy_type,
+        torch.is_complex,
```

Comment:
nit: not a big deal but `torch.Tensor.is_complex` since the list is called `_allowed_methods`? 😛

Reply:
so is_complex is only a torch function...I had tried torch.Tensor.is_complex before and it didn't work :/

Reply:
nah i am a troll, i was just using the torch equiv instead